internal/proxy/task_query.go

```diff
@@ -343,7 +364,7 @@ func (t *queryTask) PreExecute(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	t.RetrieveRequest.ReduceStopForBest = queryParams.reduceStopForBest
+	t.RetrieveRequest.ReduceType = int32(queryParams.reduceType)
```

Comment:
Maybe we shall fill both field value for backward compatibility concern?

Reply:
compatibility in the process of rolling upgrade?
this may only result in the number of results not than 'many'